### PR TITLE
Expand type check in TraceChannelInterceptor

### PR DESCRIPTION
**Pull request by CanaanStafford**
The type check in the method 'getMessage' was only for a sub-class of
MessagingException. The check was expanded to the base class.

### CODE REVIEW

**Comment by pivotal-issuemaster** [comment]
@CanaanStafford Please sign the [Contributor License Agreement](https://cla.pivotal.io/sign/spring?repositoryId=spring-cloud/spring-cloud-sleuth&pullRequestId=673)!

[Click here](https://cla.pivotal.io/sync/spring?repositoryId=spring-cloud/spring-cloud-sleuth&pullRequestId=673) to manually synchronize the status of this Pull Request.

See the [FAQ](https://cla.pivotal.io/about) for frequently asked questions.

**Comment by pivotal-issuemaster** [comment]
@CanaanStafford Thank you for signing the [Contributor License Agreement](https://cla.pivotal.io/sign/spring?repositoryId=spring-cloud/spring-cloud-sleuth&pullRequestId=673)!

File: spring-cloud-sleuth-core/src/main/java/org/springframework/cloud/sleuth/instrument/messaging/TraceChannelInterceptor.java [hunk 1]
```diff
@@ -22,10 +22,7 @@
 import org.springframework.cloud.sleuth.TraceKeys;
 import org.springframework.cloud.sleuth.Tracer;
 import org.springframework.cloud.sleuth.sampler.NeverSampler;
-import org.springframework.messaging.Message;
-import org.springframework.messaging.MessageChannel;
-import org.springframework.messaging.MessageDeliveryException;
-import org.springframework.messaging.MessageHandler;
+import org.springframework.messaging.*;
```
Comment (spencergibb) [inline spring-cloud-sleuth-core/src/main/java/org/springframework/cloud/sleuth/instrument/messaging/TraceChannelInterceptor.java]: Please don't use `*` imports.

Comment (CanaanStafford) [inline spring-cloud-sleuth-core/src/main/java/org/springframework/cloud/sleuth/instrument/messaging/TraceChannelInterceptor.java]: My apologies, I got in a hurry and completely overlooked this. Fixed. I should really figure out how to disable that in Intellij.

**Pull request merged**